docs/user/trusted-publishers/internals.md

```diff
@@ -200,6 +200,38 @@ stored ID and fails if they don't match. Through this process, only the original
 GitHub user remains authorized to publish to their PyPI projects, even if they
 change their username or delete their account.
 
+### How do I become a Trusted Publisher?
```

Comment:
Nit: we call these "Trusted Publishing providers" elsewhere:

```suggestion
### How do I become a Trusted Publishing provider?
```

(We also call them "OIDC identity provider" but I figure that's even more confusing 😅)